stdlib/public/core/DebuggerSupport.swift

```diff
@@ -179,7 +166,7 @@ public enum _DebuggerSupport {
       print(str, terminator: "", to: &target)
     }
   
-    if (maxDepth <= 0) || !willExpand {
+    if (maxDepth <= 0) {
```

Comment:
1. these parens can be removed.
2. this might be more clear as a guard?
3. why is there a `print("", ...)`? seems like that can be removed

can it be distilled to this:

```swift
guard maxDepth > 0 else {
    return
}
```

Reply:
oh I see the `print` is for a newline